src/Tesseract.Tests/EngineTests.cs

```diff
@@ -71,7 +71,7 @@ public void CanParseMultipageTifOneByOne()
         [TestCase(PageSegMode.SingleColumn, "This is a lot of 12 point text to test the")]
         [TestCase(PageSegMode.SingleLine, "This is a lot of 12 point text to test the")]
         [TestCase(PageSegMode.SingleWord, "This")]
-        [TestCase(PageSegMode.SingleChar, "T")]
+        [TestCase(PageSegMode.SingleChar, "hl")]
```

Comment:
This file actually contains an image of the letter "T". However, the margin on the right-hand side of "T" is smaller than the left, and I think that's causing the auto-thresholding algorithm to invert the thresholding and recognize the text as "hl" instead. I updated the PNG and got the expected result:
OLD:
https://github.com/charlesw/tesseract/blob/master/src/Tesseract.Tests/Data/Ocr/PSM_SingleChar.png
NEW:
![PSM_SingleChar](https://github.com/user-attachments/assets/049b95f7-f42a-4368-a087-6f888f8e5c64)

Reply:
I was planning to look into that and make sure that was a regression with the Tesseract library itself, thanks for looking into it. I'm surprised that Tesseract library would return more than once character when it's explicitly instructed to only return a single character.

Reply:
You make an interesting point about Tesseract returning 2 characters despite the PageSegMode; might be worth digging into deeper as a potential Tesseract library defect.